golang/cosmos/x/swingset/types/sim-params.go

```diff
@@ -0,0 +1,41 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// This should roughly match the values in
+// `agoric-sdk/packages/cosmic-swingset/src/sim-params.js`.
```

Comment:
Here too, and in `sim-params.js`, there should be a reference to the vocabulary of named units. Consider defining them as string constants to avoid misspelling in the code.